src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Crypto.cs

```diff
@@ -103,6 +103,20 @@ internal static string GetX509RootStoreFile()
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_CheckX509Hostname")]
         internal static extern int CheckX509Hostname(SafeX509Handle x509, string hostname, int cchHostname);
 
+        internal static byte[] GetAsn1StringBytes(SafeSharedAsn1OctetStringHandle content)
+        {
+            bool ignore = false;
+            content.DangerousAddRef(ref ignore);
+            try
+            {
+                return GetAsn1StringBytes(content.DangerousGetHandle());
+            }
+            finally
+            {
+                content.DangerousRelease();
+            }
```

Comment:
This should be modified slightly such that the add ref is inside of the try, and there should be a conditional check before the release.  Technically this doesn't matter, as it's a pattern established to handle thread aborts, which don't exist on .NET Core.  But the reason one does it that way is so that if a thread abort occurs before the add ref, the release won't happen, and if a thread abort occurs after the add ref but before entering the try block, then the release will still happen. e.g.
```C#
bool gotRef = false;
try
{
    content.DangerousAddRef(ref gotRef);
    ...
}
finally
{
    if (gotRef) content.DangerousRelease();
}
```